library.properties

```diff
@@ -1,8 +1,9 @@
-name=arduino_TLC59116
-version=1.5
-author=Alan Grover
+name=TLC59116
+version=1.8
+author=Red Byer (original by: Alan Grover)
 maintainer=Alan Grover <awgrover@msen.com>
-sentence=Library for TI TLC59116 (and almost TLC59108) I2C Led Driver chip.
+sentence=Library for TI TLC59116 (and almost TLC59108) I2C Led Driver chip. MODIFIED FOR ARDUINO 1.5+ 
 paragraph=A high-level interface that manages multiple TLC59116 chips, with almost every function of the chip implemented. And, a lower-level interface that (allegedly) correctly implements the protocol for each function without the "manager" layer.
-url=https://github.com/2splat/arduino-TLC59116
+url=https://github.com/2splat/arduino-TLC59116   This version has been modified to compile succesfully under Arduino 1.5+ (e.g. 1.8.1)
```

Comment:
trailing text-as-comment works?

Reply:
No!  Good catch. Some weirdness with my commit.  Not sure how that ended up on that line!